x-pack/plugin/apm-data/src/main/resources/ingest-pipelines/apm@pipeline.yaml

```diff
@@ -24,3 +24,13 @@ processors:
     target_field: user_agent
     ignore_failure: true
     ignore_missing: true
+- script:
+    description: Add time when event was ingested (and remove sub-seconds to improve storage efficiency)
```

Comment:
I admit i copied this from Fleet.
Do we want to adopt the same second-granularity?

Reply:
What would be the reason not to?

Reply:
Good point, I don't see any, but I also have a very limited understanding of what event.ingested is used for...

Reply:
I would choose based on what we were doing before removing it. As it was set by the Fleet pipeline truncating below the second I would keep this behavior. It's the least surprise for users.